src/content/toeic_listening_conversations_talks.py

```
@@ -0,0 +1,302 @@
+            "target": "listening_detail",
+            "scenario": "client-briefing",
+            "choices": [
+                "Reserve a meeting room.",
+                "Move a delivery appointment.",
+                "Prepare a budget report.",
+            ],
+            "choice_labels": ["A", "B", "C"],
+            "correct_index": 0,
+            "explanation": "Priya says she will book the small meeting room under Evan's name.",
+            "remediation": {
+                "front": "Comment dire que l’on réserve une salle pour quelqu’un ?",
```

Ruff (0.16.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)